_pages/support-packages.md

```diff
@@ -24,7 +24,7 @@ bg-color-class: bg-accent-warm-light
       {% for product in site.data.support-packages %}
         <div class="desktop:grid-col">
           <div class="intro">
-            {% asset "{{ product.image }}" class="square-15" %}
+            {% asset "{{ product.image }}" class="square-15" alt="" %}
```

Comment:
Why doesn't this alt have content?

Reply:
These are decorative elements, so an empty `alt` is to let screenreaders know that it can be safely skipped over.

<img width="201" alt="Screen Shot 2023-02-15 at 9 31 45 AM" src="https://user-images.githubusercontent.com/1178494/219056603-8f7b1735-e7f3-4d22-b7b9-014d55265c6c.png">

Reply:
That's fair; thanks!